tsconfig.json

```diff
@@ -9,7 +9,10 @@
     "target": "es2021",
     "module": "node16",
     "strict": true,
-    "noEmit": true
+    "noEmit": true,
+    "paths": {
+      "@slowreader/core": ["./core"]
```

Comment:
In this case we will take `core` types directly from `core/` dir without `node_modules` link.

I am not sure that it is the best solution. Both options seem very subjective for me. But I personally love this one a little more because it is 1 file compare to 2. And we need to use `@slowreader/core` in all monorepo’s projects.